app/src/main/java/com/example/coopertest/Results.kt

```diff
@@ -0,0 +1,163 @@
+package com.example.coopertest
+
+import android.content.Context
+import android.content.SharedPreferences
+import android.provider.Settings.Global.getString
+import androidx.preference.PreferenceManager
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+
+class Results (meters : Double, context: Context){
+
+    var meters : Double=0.0
+    var age: Int=0
+    var athlete: Boolean = false
+    var gender : String = ""
```

Comment:
Gender in our app could also be boolean, or even better enum, but it's better not to use plain String

Reply:
Yes, we can, but it's more for ethical problem: where you put 0 and where you put 1 : some persons can make a big debate :D

Reply:
Yes, thats why enum is best :D

Reply:
And you did that (assigning numbers) in resources anyway from what I've seen earlier :laughing:

Reply:
Yes because at the beginning I created Boolean, and after I thought it was not a good idea, but I forgot to remove from arrays